src/File.php

```diff
@@ -160,7 +163,24 @@ class File extends DataObject implements AssetContainer, Thumbnail, CMSPreviewab
     ];
 
     private static $indexes = [
-        'FileHash' => true
+        'FileHash' => true,
+        // Used when viewing files in AssetAdmin ordered by Title ASC
+        // Not used for other sort (e.g. Title DESC or Created) so if the default
+        // sort in AssetAdmin changes, we should update this index.
+        'default_asset_sort' => [
+            'type' => 'index',
+            'columns' => [
+                'ParentID',
+                'IsFolder DESC',
+                'Title',
+            ],
+        ]
+    ];
+
+    private static bool|array $skip_afterwrite_generatedcolumn_fetch = [
```

Comment:
```suggestion
    private static bool|array $skip_fetch_generated_columns_after_write = [
```

Reply:
I was hoping for it to get shorter lol. This is still better than what I had so I'll make the change :+1: